content/docs/concepts/accountability/_index.md

```diff
@@ -0,0 +1,385 @@
+  - `history`: assigned the count of proven faults committed by the offending validator
+  - `epoch offences count`: assigned the count of proven faults created by all validators in the epoch
+- `slashing rate` is computed:
+  - `base rate + epoch offences count * collusion factor + history * history factor`.
+- `slashing amount` of the fine is computed:
+  - `(slashing rate * validator bonded stake)/slashing rate precision`
+- the slashing is computed:
+  - the slashed amount of NTN stake token is subtracted from the validator’s bonded stake and transferred to the Autonity Protocol global `treasury` account for community funding
+  - the slashing fine is applied to validator bonded stake according to the protocol’s [Penalty Absorbing Stake (PAS)](/concepts/accountability/#penalty-absorbing-stake-pas) model
+  - the `jail period` of the validator is computed to determine the validator's jail release block number: `current block number + jail factor * history * epoch period`.
+- the validator state is updated: (a) the self-bonded and total staked amounts, (b) the slashing amount is added to the validator's `totalSlashed` amount.
+
+### Penalty Absorbing Stake (PAS) 
+
+Autonity implements a [_penalty absorbing stake_](glossary/#penalty-absorbing-stake-pas) model where validator [self-bonded](glossary/#self-bonded) stake is slashed in priority to [delegated](glossary/#delegated) stake when applying slashing penalties for accountability events.
+
+Slashing priority is simply:
+
+- self-bonded stake is slashed as first priority until exhausted
+- delegated stake is slashed as second priority when the slashing amount exceeds the amount of self-bonded stake available.
+
+Consequently, validator operator stake ("skin in the game") is at first risk and provides loss absorbing capital in the case of a slashing event. Self-bonding stake is not just a source of income but a public commitment to the operational integrity of a validator node by a validator operator. Potential stake delegators can use the amount of self-bonded stake of a validator as a decision factor when conducting due diligence before staking.
+
+## Rules
+
+### Accountability rules
+
+Accountability rules are applied to detect faults in the three Tendermint consensus round phases *propose*, *prevote*, *precommit*. Rules are detailed in the table below.
+
+{{% alert title="Note" %}}
+In the table:
+
+- *value*: is shorthand for "block proposal"
+- Rule ID: is the unique identifier for the Rule defined in the AFD rule engine. ID prefixes correspond to Tendermint consensus phases:
+  - `PO`: *propose*
+  - `PV`: *prevote*
+  - `C`: *precommit*
+{{% /alert %}}
+
+| Rule ID | Description |
+| --| --|
+| `PN` | Proposer has proposed a new *value* (i.e. block proposal) having (a) already sent a precommit message earlier, or (b) has received sufficiently many prevotes for an earlier proposal *value* during the same round. |
```

Comment:
- Point a) we need to specify that we have a fault only if the precommit is not-nil --> "already sent a **non-nil** precommit ...". Proposing a new value after a nil precommit is allowed.
- point b) is theoretically correct but it is not implemented in the code as it is not _observable_. Meaning we cannot know which messages a certain node has received (unless we are that node). Thus the only check implemented in the code is the one for the precommits. I would be for removing the quorum prevotes part since this is code documentation. I think the prevotes part belongs to a more theoretical specification, but I might be wrong.

Reply:
OK, so, if I edit (a) and comment out (b) then it becomes:

| Rule ID | Description |
| --| --|
| `PN` | Proposer has proposed a new *value* (i.e. block proposal) having already sent a _non-nil_ precommit message earlier <!-- , or (b) has received sufficiently many prevotes for an earlier proposal *value* during the same round. --> |

Reply:
OK, so, if I edit (a) and comment out (b) then it becomes:

| Rule ID | Description |
| --| --|
| `PN` | Proposer has proposed a new *value* (i.e. block proposal) having already sent a _non-nil_ precommit message earlier <!-- , or (b) has received sufficiently many prevotes for an earlier proposal *value* during the same round. --> |